File: app/src/main/java/com/codeartist/androidpractice/PropertyApplication.java
package com.codeartist.androidpractice;

import android.app.Application;

import com.codeartist.androidpractice.repository.Repository;

public class PropertyApplication extends Application {
    public Repository repository;
    @Override
    public void onCreate() {
        super.onCreate();
        //repository = Repository.getInstance(this);
    }

}

File: app/src/main/java/com/codeartist/androidpractice/PropertyApplication.kt
package com.codeartist.androidpractice

import android.app.Application
import com.codeartist.androidpractice.db.PropertyRoomDatabase
import com.codeartist.androidpractice.repository.PropertyRepository
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp
class PropertyApplication : Application() {
}